tests/conftest.py

```diff
@@ -194,88 +194,88 @@ def clean_client(
 
 @pytest.fixture
 def sql_store() -> BaseZenStore:
-    with tempfile.TemporaryDirectory(suffix="_zenml_sql_test") as temp_dir:
-        store = SqlZenStore(
```

Comment:
For my understanding, what was wrong with the context? It was deleting the temporary files while they were still needed?

Reply:
Would be curious as well

Reply:
the SQLite store database created in the temporary directory is still in use when the tests complete. Windows doesn't like if you're trying to remove open files. Without the context, the temp directory is left behind.

I vaguely remember you facing a similar issue with using this context approach in materializers.